modules/test/helpers/configmap.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// GetConfigMap -
+func GetConfigMap(name types.NamespacedName) corev1.ConfigMap {
+	cm := &corev1.ConfigMap{}
+	gomega.Eventually(func(g gomega.Gomega) {
+		g.Expect(k8sClient.Get(ctx, name, cm)).Should(gomega.Succeed())
+	}, timeout, interval).Should(gomega.Succeed())
```

Comment:
timeout should probably be a parameter that each caller can have its own timetout/interval

Reply:
https://github.com/openstack-k8s-operators/lib-common/pull/76/commits/1a42a1fc9cf2a9b49b969dea1c6c1660647abe5c